### ISSUE TITLE
Color contrast insufficient in several areas

### ISSUE THREAD

**Issue by robfentress** [ucfopen/UDOIT#173]
Color contrast for `.badge-error` and `.btn-success` is insufficient, as is the white on grey text of the labels for panels.  I'm not sure where that is in the code, but it is in the upper left hand corner of the panels.

Here is the preview presented by the [Color Contrast Analyzer Chrome extension](https://chrome.google.com/webstore/detail/color-contrast-analyzer/dagdlcijhfbmgkjokkjicnnfimlebcll?hl=en), with 3:1 contrast ratio being tested for.  Anywhere where there isn't an outline around the text, there is insufficient contrast:

![contrast-wcag-aa-large](https://cloud.githubusercontent.com/assets/9356499/16451647/4bef6a30-3dd2-11e6-84a2-884c3c1cbfe8.png)
